pkg/customresource/repositoryserver.yaml

```diff
@@ -42,6 +42,8 @@ spec:
                       size details that can be used while establishing connection
                       to the kopia repository
                     properties:
+                      cacheDirectory:
+                        type: string
```

Comment:
Same thing here as well, we should move this to some place else since it does not make sense under `cacheSizeSettings`.